package.json

```diff
@@ -14,13 +14,13 @@
     "./packages/*"
   ],
   "devDependencies": {
-    "@changesets/cli": "2.26.2",
-    "prettier": "^3.0.3",
-    "prettier-plugin-tailwindcss": "^0.5.4"
+    "@changesets/cli": "2.27.1",
+    "prettier": "^3.2.5",
+    "prettier-plugin-tailwindcss": "^0.5.12"
   },
   "pnpm": {
     "patchedDependencies": {
-      "monaco-editor@0.43.0": "patches/monaco-editor@0.43.0.patch"
+      "monaco-editor@0.47.0": "patches/monaco-editor@0.47.0.patch"
```

Comment:
I made a new patch here for monaco-editor, so the bundle size stay small ( only bundle js, css, html ,ts )